modules/access-management/variables.tf

```diff
@@ -0,0 +1,1535 @@
+##############################################################################
+# Admin Observability Variables
+##############################################################################
+
+variable "admin_observability_ag_name" {
+  type        = string
+  description = "Name of the Administrator Observability access group"
+  default     = "admin-observability-group"
+}
+
+variable "admin_observability_ag_policies" {
+  type = map(object({
+    roles              = list(string)
+    account_management = optional(bool)
+    tags               = set(string)
+    resources = optional(list(object({
+      region               = optional(string)
+      attributes           = optional(map(string))
+      service              = optional(string)
+      resource_instance_id = optional(string)
+      resource_type        = optional(string)
+      resource             = optional(string)
+      resource_group_id    = optional(string)
+    })))
+    resource_attributes = optional(list(object({
+      name     = string
+      value    = string
+      operator = optional(string)
+    })))
+  }))
+  description = "A map of policies for the Administrator Observability access group, has a set of default policies that can be overridden"
+  default = {
+    "atracker" = {
+      roles = ["Administrator", "Writer"]
+      resources = [{
+        service = "atracker"
+      }]
+      tags = []
+    }
+    "observability" = {
+      roles = ["Administrator", "Manager"]
+      resources = [{
+        service = "sysdig-monitor"
+      }]
+      tags = []
+    }
+    "logdna" = {
+      roles = ["Administrator", "Manager"]
+      resources = [{
+        service = "logdna"
+      }]
+      tags = []
+    }
+  }
+}
+
+variable "admin_observability_ag_dynamic_rules" {
+  type = map(object({
+    expiration        = number
+    identity_provider = string
+    conditions = list(object({
+      claim    = string
+      operator = string
+      value    = string
+    }))
+  }))
+  description = "A map of dynamic rules for the admin observability access group"
+  default     = {}
+}
+
+variable "admin_observability_ag_add_members" {
+  type        = bool
+  description = "Enable this to add members to the admin observability group"
+  default     = false
+}
+
+variable "admin_observability_ag_description" {
+  type        = string
+  description = "Description of the admin observability access group"
+  default     = null
+}
+
+variable "admin_observability_ag_tags" {
+  type        = list(string)
+  description = "Tags that should be applied to the service"
```

Comment:
`The list of tags that you want to associated with your admin observability access group` maybe? same for the others